nebari-dev/nebari
# Precommit typos

## Reference Issues or PRs

Closes https://github.com/nebari-dev/nebari/issues/2598



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?

1. Create a new [local deployment](https://www.nebari.dev/docs/how-tos/nebari-local) from the `develop` branch in editable mode
2. Run `nebari render -c nebari-config.yaml`
3. Run `ls stages/07-kubernetes-services/modules/kubernetes/services/dask-gateway/` and verify that the directory contains the misspelled `controler.tf` file.
4. Checkout this branch and run `nebari render -c nebari-config.yaml`
5. Re-run the `ls` from above and verify the file has been renamed to `controller.tf`



## Any other comments?
